doc/changelog/08-tools/10577-extraction-dependent-projections.rst

```diff
@@ -0,0 +1,8 @@
+- Fix a printing bug of OCaml extraction on dependent record projections, which
+  produced improper `assert false`. This change makes the OCaml extractor
+  internally inlines record projections by default and may not produce record
+  projection constants anymore. If you want to extract record projection
+  constants, you may need to instruct it explicitly to the extractor
```

Comment:
I am not sure how obvious this is to people who are used to extraction, but if it is not you might want to say using which command.

Reply:
Thank you for suggestions. I have indicated that the change of behavior is specific to the monolithic OCaml extraction (`Extraction "file"` and `Recursive Extraction`).